pkg/pillar/types/zedroutertypes.go

```diff
@@ -1870,6 +1871,37 @@ type NetworkXObjectConfig struct {
 type IpRange struct {
 	Start net.IP
 	End   net.IP
+	Size  int
+}
+
+// Contains used to evaluate whether an IP address
+// is within the range
+func (ipRange IpRange) Contains(ipAddr net.IP) (bool, error) {
+	start := ipRange.Start.To4()
+	end := ipRange.End.To4()
+	cur := ipAddr.To4()
+	if cur == nil || start == nil || end == nil {
+		errStr := fmt.Sprintf("invalid configuration (%s, %s, %s)\n",
+			ipAddr.String(), ipRange.Start.String(), ipRange.End.String())
+		return false, errors.New(errStr)
+	}
```

Comment:
Why exclude IPv6? 

This Compare function doesn't need to return an error.
It can directly do 
return bytes.Compare(ipAddr, ipRange.Start) >= 0 &&
		bytes.Compare(ipAddr, ipRange.End) <= 0